webdriver/util/cleanup.py

```diff
@@ -0,0 +1,47 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
```

Comment:
I wrote this code. I hereby relinquish any rights I have on the code to the WPT project default license.